test/FunctionsV1/WebJobs.Extensions.DurableTask.Tests.V1.csproj

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="4.19.4" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.6.3" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="2.3.0" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="2.2.0" />
```

Comment:
Was this change intentional?

Reply:
Yes, there was a warning in the test project because the src code uses 2.2.0.

Reply:
Could go in a separate PR if we want to keep this commit purely what is described in it.

Reply:
Oh - I didn't realize this was the test project. I have no problem with taking this change in the current PR.